### DISCUSSION TITLE
Traceback error

### DISCUSSION THREAD

**Discussion by MoniBoosDevotedBoyfriend** [Monika-After-Story/MonikaModDev#9572] [QA - tech support + general] [answered]
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/9620600/traceback.txt)
Hey guys. Traceback question for you all. Do I only need to fix the promise ring file, or is there something more to this?

**Comment by Booplicate** [comment, marked as answer]
You can just update to the latest version.

**Reply by MoniBoosDevotedBoyfriend** [reply]
I mean, I already fixed the file itself, so it should be fine regardless right?

**Reply by Booplicate** [reply]
I don't want to guess, so I suggest to update.

**Reply by MoniBoosDevotedBoyfriend** [reply]
Alright. Sounds good

**Reply by MoniBoosDevotedBoyfriend** [reply]
Btw I didn’t get to update to 12.10. It’s taking me straight to 12.11 will that be fine?

**Reply by MoniBoosDevotedBoyfriend** [reply]
Sorry, I’m super paranoid

**Reply by Booplicate** [reply]
Yes, it's how it should be.

**Reply by MoniBoosDevotedBoyfriend** [reply]
Ok. One last thing. Will I still be able to use my custom ring files to replace the default ones?

**Reply by Booplicate** [reply]
You can just override the image name to use instead of overwriting files every update.
But yes, technically you can do that.

**Reply by MoniBoosDevotedBoyfriend** [reply]
Yeah no that’s fine, I just wanted to make sure it’s still safe to use my own custom ones.

**Reply by MoniBoosDevotedBoyfriend** [reply]
Ok. I updated, added the rings and everything seems to be ok. Didn’t get an error message (although I did JUST open the game again, but it should be fine regardless). So, thank you for the help

**Comment by MoniBoosDevotedBoyfriend** [comment]
On a similar note, there was another traceback file that had something to do with something spaceroom. I’m pretty confident it was one of the spaceroom image files, but I could be wrong. I’m assuming it was fixed by the update?
